src/base/playback.js

```diff
@@ -210,6 +221,25 @@ export default class Playback extends UIObject {
   destroy() {
     this.remove()
   }
+
+  /**
+   * attempt to autoplays the playback.
+   * @method autoPlay
+   */
+  autoPlay() {
+    this.canAutoPlay((result, error) => { // eslint-disable-line no-unused-vars
+      result && this.play()
```

Comment:
Here we could maybe log the error as warning message ?

Reply:
I agree, there should be a warning message whenever a non-fatal error occurs.

Reply:
I added a Log.warn() in 6e6df23e35c49d67a68bb281790010b4c5dd7bcc